src/bio/bio_internal.h

```diff
@@ -28,7 +28,8 @@
 #include <daos_srv/bio.h>
 
 #define BIO_DMA_PAGE_SHIFT	12		/* 4K */
-#define	BIO_DMA_PAGE_SZ		(1UL << BIO_DMA_PAGE_SHIFT)
+#define BIO_DMA_PAGE_SZ		(1UL << BIO_DMA_PAGE_SHIFT)
+#define BIO_XS_CNT_MAX		48
```

Comment:
a few comment is helpful

Reply:
Sure, I'll add more comment if the patch needs be refreshed, or I'll add it in follow-on patch.